### PR TITLE
 Enhance styling of navigation text ⚡

### PR DESCRIPTION
**Pull request by Nishitbaria**
## Related Issue*
https://github.com/GameSphere-MultiPlayer/GameSphere/issues/543

Closes #issue_number

## Description

This pull request aims to improve the visibility and clarity of the navigation text by adding styling enhancements. The following changes have been made:

1. Increased visibility and clarity of text:

- Set the text color to white for improved visibility.
- Added a subtle text shadow to enhance readability.
- Increased the font weight for better emphasis.
- Hover effect:

2. Hover effect:

- Added a hover effect that changes the text color to a bright red (#ff0000) when hovering over the navigation links.
- This provides visual feedback to users when interacting with the navigation.

## Screenshot Section

`before`

image

`after`

image

## Checklist 

- [x] I have added image into screensort in "SinglePlayer - Games/Banner - image" in this section*
- [x] I have added game into "additionalpage/game.html"*
- [x] I have added Update main README.md file*
- [x] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

### CODE REVIEW

**Comment by Nishitbaria** [comment]
@Durgesh4993  Please Review my PR

**Comment by Durgesh4993** [comment]
@Nishitbaria Resolve the conflict

**Comment by Nishitbaria** [comment]
Hey **@Durgesh4993** Please Review PR and  See now all conflicts are solved

File: dist/main.css
Comment (Durgesh4993) [inline dist/main.css:1]: https://github.com/GameSphere-MultiPlayer/GameSphere/assets/56786344/375fad6d-1b72-4f3d-86ab-fb55fa3497e7

**Pull request merged**